playwright.config.ts

```diff
@@ -21,6 +21,7 @@ export default defineConfig({
   /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
   use: {
     /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
+    headless: true,
```

Comment:
so if this will run the tests without the interface maybe update the documentation? To say `if you would like to run locally with the interface update "headless" to false` or sth like that

Reply:
when I use `npm run test:e2e` it stil opens, to be fair this can be removed since it was a suggestion from playwright when test was failing. Removing now

Reply:
ah ok then it's fine I guess I though it would not open when running `npm run test:e2e`

Reply:
however, it has been removed